### PR TITLE
Update default wrapper

### LINKED ISSUE #4651: Create ShareProgress Page Wrapper
Design Issue: https://github.com/mozillafoundation/design/issues/517

Create wrapper page HTML <style> to reflect new branded wrapper design.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/4651

This has passed [design review](https://github.com/MozillaFoundation/Design/issues/517#issuecomment-635001330).

Test page: http://share.mozilla.org/352/207160

---

@pomax I've committed the current wrapper code to `master` so you can see the code difference in this PR. I had to add a bunch `!important` rules to override the very dated CSS code Shareprogress injects to the page. There are quite a bit of tech limitations with Shareprogress that's why simple styling changes ended up requiring a ton of CSS overrides on our end 🙈 .

### CODE REVIEW

File: default-wrapper.html [hunk 1]
```diff
@@ -10,68 +10,26 @@
   <meta name="viewport"
         content="width=device-width, initial-scale=1, maximum-scale=1">
   <link href="/basic.css" media="all" rel="stylesheet" type="text/css" />
```
Comment (Pomax) [inline default-wrapper.html]:
```suggestion
  <link href="/basic.css" media="all" rel="stylesheet">
```

File: mofo-shareprogress-page-wrapper.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+.cta {
+  font-size: 26px;
+  font-weight: 200;
+  line-height: 35.75px;
+  margin-bottom: 10px;
+  margin-top: 20px;
+  text-align: center;
+  margin: 0 auto;
+  font-family: "ZillaSlab-Light", "Arvo","Fira Sans", "Helvetica Neue", "Arial", sans-serif;
```
Comment (Pomax) [inline mofo-shareprogress-page-wrapper.css]: we don't load any of those other fonts, probably better to cut out Arvo and Fira Sans? Or add their webfont `<link>`

File: mofo-shareprogress-page-wrapper.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
+/* All the !important rules are
+   To override CSS rules injected by Shareprogress' script */
+
+body {
+  background: #b7b9fa; /* Foudation Brand $light-blue */
+  font-family: 'Nunito Sans', sans-serif !important;
```
Comment (Pomax) [inline mofo-shareprogress-page-wrapper.css:6]: we do not have a google font `<link>` for Nunito

**Review by Pomax — CHANGES_REQUESTED** [review]

File: mofo-shareprogress-page-wrapper.css (hunk 3 above)
Comment (mmmavis) [reply mofo-shareprogress-page-wrapper.css:6]: We do. In `default-wrapper.html` line 13.
```html
<link href="https://fonts.googleapis.com/css2?family=Nunito+Sans&family=Zilla+Slab:wght@400;500&display=swap" rel="stylesheet">
```

Though I need to double-check if I include the right font weights 😂

File: mofo-shareprogress-page-wrapper.css (hunk 2 above)
Comment (mmmavis) [reply mofo-shareprogress-page-wrapper.css]: Ah good call. This is actually from the existing code. This entire chunk might not be relevant anymore. Will double check.

File: mofo-shareprogress-page-wrapper.css (hunk 3 above)
Comment (Pomax) [reply mofo-shareprogress-page-wrapper.css:6]: I am <s> 👀</s>

File: mofo-shareprogress-page-wrapper.css [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+.sprog-content #main_share .email-share .email-btn,
+.sprog-content #main_share .fb-share .fb-btn {
+  width: 100% !important;
+  font-family: 'Nunito Sans', sans-serif !important;
+}
+
+.sprog-content #main_share .email-share .email-btn a,
+.sprog-content #main_share .fb-share .fb-btn a {
+  background: #595cf3 !important; /* Foudation brand $blue */
```
Comment (Pomax) [inline mofo-shareprogress-page-wrapper.css]:
```suggestion
  background: #595cf3 !important; /* Foundation brand $blue */
```

**Review by Pomax — APPROVED** [review]

**Comment by mmmavis** [comment]
Yay!

**Pull request merged**